sky/packages/sky/lib/src/widgets/snack_bar.dart

```diff
@@ -38,7 +38,7 @@ class SnackBarAction extends Component {
   }
 }
 
-class SnackBar extends Component {
+class SnackBar extends StatefulComponent {
```

Comment:
I agree that it is unfortunate for SnackBar. I think it's worth noting though that SnackBar is more or less the only case where it made things worse. The common case does appear to be owning the performance and linking many transitions to it.

Having said that, I think we can make this better again. Will think on it more.